### PR TITLE
Fix #133: Move Sense2Vec as an Optional Dependency

### LINKED ISSUE #133: move sense2vec to an optional dependency
I feel like spaCy is still very much a relevant dependency to have, but sense2vec is a bit niche. Seems like a valid candidate to move to an optional dep.

### PR DESCRIPTION
**Pull request by koaning**
https://github.com/RasaHQ/whatlies/issues/133

### CODE REVIEW

**Comment by koaning** [comment]
This makes the base package lighter.

File: setup.py [hunk 1]
```diff
@@ -79,6 +82,7 @@
         "dev": dev_packages,
         "test": test_packages,
         "tfhub": tf_packages,
+        "sense2vec": tf_packages,
```
Comment (mkaze) [inline setup.py:85]: You probably meant writing `s2v_packages` here ;)

**Comment by koaning** [comment]
I think this branch is now clean. @mkaze any comments?

**Comment by mkaze** [comment]
@koaning No comments! Looks good to me.

**Pull request merged**